### DISCUSSION TITLE
Ability to Disable GraphQL

### DISCUSSION THREAD

**Discussion by lu-zen** [directus/directus#4954] [Archive]
I wanna use REST only, how can I disable graphql ?

**Comment by rijkvanzanten** [comment]
There's currently no way to enable/disable rest/GraphQL

**Reply by WoLfulus** [reply]
As a workaround: if you're running behind a load balancer/reverse proxy, force a 404 (or 403) response to any request under /graphql*

(this doesn't apply to REST since it's **required** by the app)
